docs/agents/sematext-agent/containers/events.md

```diff
@@ -0,0 +1,21 @@
+title: Container events
+description: Explains how to control the event stream collector and available filters
+
+Sematext Agent collects all sorts of container life-cycle, image, daemon, volume, plugin, and network events. If you're not interested in a certain event category, you can instruct the agent to ignore events that pertain to such a category via the `CONTAINER_EXCLUDED_EVENTS` environment variable. For example, to drop the volume and image events you would define the following environment variable:
```

Comment:
Should be `CONTAINER_EVENTS_EXCLUDED` ????

Reply:
Yes, good catch :)